python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/aaa-server-groups.j2

```diff
@@ -4,22 +4,18 @@
  that can be found in the LICENSE file.
 #}
 {# eos - aaa server groups #}
-{% if aaa_server_groups is arista.avd.defined %}
-{%     for aaa_server_group in aaa_server_groups %}
-{%         if aaa_server_group.type is arista.avd.defined and aaa_server_group.name is arista.avd.defined %}
+{% for aaa_server_group in aaa_server_groups | arista.avd.natural_sort('name') | arista.avd.natural_sort('type') %}
```

Comment:
radius and tacacs comes together and in order of type also but ldap not reflect along with radius and tacacs. ldap and radius/tacacs has some other config in between so we might to create another module for ldap.
```
hostname wan2-dc1
ip name-server vrf MGMT 10.14.0.1
dns domain fun.aristanetworks.com
!
aaa group server ldap ZZZZ
   server 12.1.1.1
   server 10.1.1.1 vrf mahesh
!
router adaptive-virtual-topology
   topology role edge
   region Global id 1
   zone Global-ZONE id 1
   site DC1 id 100
```  

```
management api http-commands
   no shutdown
   !
   vrf MGMT
      no shutdown
!
aaa group server radius XYZ
   server 10.1.1.1 vrf mahesh
   server 11.1.1.1 vrf kumar
   server 12.1.1.1
   server 10.10.10.157 vrf mgt
   server 10.10.10.249
!
aaa group server tacacs+ TACACS2
   server 192.168.10.157 vrf mgt
   server 10.10.10.248
   server 10.1.1.1
!
aaa authorization exec default local
```

Reply:
Seperated LDAP group server from aaa-group-server template.